init/main.go

```diff
@@ -697,6 +705,57 @@ func checkIfInitrd() error {
 	return nil
 }
 
+func blkInfoMatchesArg(info *blkInfo, arg string) bool {
+	if strings.HasPrefix(arg, "UUID=") && info.uuid == strings.TrimPrefix(arg, "UUID=") {
+		return true
+	}
+	if strings.HasPrefix(arg, "LABEL=") && info.label == strings.TrimPrefix(arg, "LABEL=") {
+		return true
+	}
+
+	return false
+}
+
+func resume(devpath string) error {
```

Comment:
Is there anything need to be done by booster at suspend time? Or it is assumed that the host software (e.g. systemd) handles it without involving booster.

Reply:
nope, as long as `/sys/power/resume` and `/sys/power/resume_offset` are set correctly, when suspend-to-disk happens the kernel handles everything